chromium_src/net/dns/dns_transaction.cc

```diff
@@ -0,0 +1,49 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "base/strings/string_util.h"
+#include "brave/components/unstoppable_domains/buildflags/buildflags.h"
+#include "net/dns/dns_config.h"
+#include "net/dns/dns_server_iterator.h"
+
+#if BUILDFLAG(UNSTOPPABLE_DOMAINS_ENABLED)
+#include "brave/components/unstoppable_domains/constants.h"
```

Comment:
I've just understood that `src/net` cannot depend on `src/components` :(

Reply:
So maybe we should push `constants` and `utils` down to the `net` layer

Reply:
I've pulled necessary constants into net layer in 5f9dc0f3cafa00174ff3872a74e14d1c2b15d561.
I did not move utils but remove the usage of `IsUnstoppableDomainsEnabled` function at https://github.com/brave/brave-core/pull/7702/commits/5f9dc0f3cafa00174ff3872a74e14d1c2b15d561#diff-3ce0a287b7804c3416893f38bed22d098d3c8756b659ca5915ca088bb674434eR18, kDoHResolver is a resolver specific tied to this feature (with that {brave_UD} suffix), and we don't want to use it for non-crypto domains in any case.